
<h3>Total Reads</h3>
<table><tr><th>Sample</th><th>Note</th>
<th>Total Purity Filtered Reads Sequenced</th><th>Alternative Aligments</th><th>Failed Vendor QC Check</th><th>Read Length</th><th>Estimated Library Size</th>
</tr>
<tr><td>CTRL-rep1</td><td>NA</td><td align='right'>35,277,950</td><td align='right'>3,549,352</td><td align='right'>NA</td><td align='right'>101</td><td align='right'>89,355,723</td></tr>
<tr><td>CTRL-rep2</td><td>NA</td><td align='right'>37,176,382</td><td align='right'>3,581,878</td><td align='right'>NA</td><td align='right'>101</td><td align='right'>88,452,046</td></tr>
<tr><td>CTRL-rep3</td><td>NA</td><td align='right'>36,126,708</td><td align='right'>3,579,076</td><td align='right'>NA</td><td align='right'>101</td><td align='right'>85,737,150</td></tr>
<tr><td>ESRP-DKO-rep1</td><td>NA</td><td align='right'>38,211,590</td><td align='right'>4,160,502</td><td align='right'>NA</td><td align='right'>101</td><td align='right'>87,565,896</td></tr>
<tr><td>ESRP-DKO-rep2</td><td>NA</td><td align='right'>40,854,932</td><td align='right'>4,193,544</td><td align='right'>NA</td><td align='right'>101</td><td align='right'>86,703,703</td></tr>
<tr><td>ESRP-DKO-rep3</td><td>NA</td><td align='right'>39,939,930</td><td align='right'>4,872,056</td><td align='right'>NA</td><td align='right'>101</td><td align='right'>83,527,470</td></tr>
</table>

<p><b>Total Purity Filtered Reads Sequenced</b> are filtered for vendor fail flags and exclude alternative alignment reads. <b>Alternative Aligments</b> are duplicate read entries providing alternative coordinates. <b>Failed Vendor QC Check</b> are reads which have been designated as failed by the sequencer. <b>Read Length</b> is the maximum length found for all reads. <b>Estimated Library Size</b> is the number of expected fragments based upon the total number of reads and duplication rate assuming a Poisson distribution.</p>

<h3>Mapped Reads</h3>
<table><tr><th>Sample</th><th>Note</th>
<th>Mapped</th><th>Mapping Rate</th><th>Mapped Unique</th><th>Mapped Unique Rate of Total</th><th>Unique Rate of Mapped</th><th>Duplication Rate of Mapped</th><th>Base Mismatch Rate</th><th>rRNA</th><th>rRNA rate</th>
</tr>
<tr><td>CTRL-rep1</td><td>NA</td><td align='right'>34,108,792</td><td align='right'>0.967</td><td align='right'>31,051,364</td><td align='right'>0.880</td><td align='right'>0.910</td><td align='right'>0.090</td><td align='right'>0.002</td><td align='right'>43,626</td><td align='right'>0.001</td></tr>
<tr><td>CTRL-rep2</td><td>NA</td><td align='right'>36,052,268</td><td align='right'>0.970</td><td align='right'>32,615,966</td><td align='right'>0.877</td><td align='right'>0.905</td><td align='right'>0.095</td><td align='right'>0.002</td><td align='right'>55,248</td><td align='right'>0.001</td></tr>
<tr><td>CTRL-rep3</td><td>NA</td><td align='right'>35,139,908</td><td align='right'>0.973</td><td align='right'>31,773,182</td><td align='right'>0.879</td><td align='right'>0.904</td><td align='right'>0.096</td><td align='right'>0.002</td><td align='right'>36,025</td><td align='right'>0.001</td></tr>
<tr><td>ESRP-DKO-rep1</td><td>NA</td><td align='right'>36,574,836</td><td align='right'>0.957</td><td align='right'>33,008,206</td><td align='right'>0.864</td><td align='right'>0.902</td><td align='right'>0.098</td><td align='right'>0.002</td><td align='right'>37,177</td><td align='right'>0.001</td></tr>
<tr><td>ESRP-DKO-rep2</td><td>NA</td><td align='right'>39,661,626</td><td align='right'>0.971</td><td align='right'>35,452,834</td><td align='right'>0.868</td><td align='right'>0.894</td><td align='right'>0.106</td><td align='right'>0.002</td><td align='right'>40,644</td><td align='right'>0.001</td></tr>
<tr><td>ESRP-DKO-rep3</td><td>NA</td><td align='right'>38,378,336</td><td align='right'>0.961</td><td align='right'>34,288,974</td><td align='right'>0.859</td><td align='right'>0.893</td><td align='right'>0.107</td><td align='right'>0.002</td><td align='right'>50,565</td><td align='right'>0.001</td></tr>
</table>

<p><b>Mapped</b> reads are those that were aligned. <b>Mapping Rate</b> is per total reads. <b>Mapped Unique</b> are both aligned as well as non-duplicate reads. <b>Mapped Unique Rate of Total</b> is per total reads. <b>Unique Rate of Mapped</b> are unique reads divided by all mapped reads. <b>Duplication Rate of Mapped</b> is the duplicate read divided by total mapped reads.  <b>Base Mismatch Rate</b> is the number of bases not matching the reference divided by the total number of aligned bases. <b>rRNA</b> reads are non-duplicate and duplicate reads aligning to rRNA regions as defined in the transcript model definition. <b>rRNA Rate</b> is per total reads. </p>

<h3>Mate Pairs</h3>
<table><tr><th>Sample</th><th>Note</th>
<th>Mapped Pairs</th><th>Unpaired Reads</th><th>End 1 Mapping Rate</th><th>End 2 Mapping Rate</th><th>End 1 Mismatch Rate</th><th>End 2 Mismatch Rate</th><th>Fragment Length Mean</th><th>Fragment Length StdDev</th><th>Chimeric Pairs</th>
</tr>
<tr><td>CTRL-rep1</td><td>NA</td><td align='right'>17,049,941</td><td align='right'>NA</td><td align='right'>0.967</td><td align='right'>0.967</td><td align='right'>0.002</td><td align='right'>0.002</td><td align='right'>139</td><td align='right'>165</td><td align='right'>45</td></tr>
<tr><td>CTRL-rep2</td><td>NA</td><td align='right'>18,021,217</td><td align='right'>NA</td><td align='right'>0.970</td><td align='right'>0.970</td><td align='right'>0.002</td><td align='right'>0.002</td><td align='right'>138</td><td align='right'>162</td><td align='right'>32</td></tr>
<tr><td>CTRL-rep3</td><td>NA</td><td align='right'>17,566,149</td><td align='right'>NA</td><td align='right'>0.973</td><td align='right'>0.973</td><td align='right'>0.002</td><td align='right'>0.002</td><td align='right'>140</td><td align='right'>164</td><td align='right'>49</td></tr>
<tr><td>ESRP-DKO-rep1</td><td>NA</td><td align='right'>18,282,375</td><td align='right'>NA</td><td align='right'>0.957</td><td align='right'>0.957</td><td align='right'>0.002</td><td align='right'>0.002</td><td align='right'>140</td><td align='right'>170</td><td align='right'>102</td></tr>
<tr><td>ESRP-DKO-rep2</td><td>NA</td><td align='right'>19,826,126</td><td align='right'>NA</td><td align='right'>0.971</td><td align='right'>0.971</td><td align='right'>0.002</td><td align='right'>0.002</td><td align='right'>142</td><td align='right'>166</td><td align='right'>60</td></tr>
<tr><td>ESRP-DKO-rep3</td><td>NA</td><td align='right'>19,183,630</td><td align='right'>NA</td><td align='right'>0.961</td><td align='right'>0.961</td><td align='right'>0.002</td><td align='right'>0.002</td><td align='right'>142</td><td align='right'>168</td><td align='right'>187</td></tr>
</table>

<p><b>Mapped Pairs</b> is the total number of pairs for which both ends map. <b>Unpaired Reads</b> are the number of reads that are lacking a mate. <b>End 1/2 Mapping Rate</b> is the number of mapped divided by the total number of End1/End2 reads. <b>End 1/2 Mismatch Rate</b> is the number of End 1 and 2 bases not matching the reference divided by the total number of mapped End 1 and 2 bases. <b>Fragment Length Mean/StdDev</b> is the mean distance, standard deviation between the start of an upstream read and the end of the downstream one. Only fragments contained within single exons are used. <b>Chimeric Pairs</b> are pairs whose mates map to different genes. </p>

<h3>Transcript-associated Reads</h3>
<table><tr><th>Sample</th><th>Note</th>
<th>Intragenic Rate</th><th>Exonic Rate</th><th>Intronic Rate</th><th>Intergenic Rate</th><th>Split Reads</th><th>Expression Profiling Efficiency</th><th>Transcripts Detected</th><th>Genes Detected</th>
</tr>
<tr><td>CTRL-rep1</td><td>NA</td><td align='right'>0.970</td><td align='right'>0.870</td><td align='right'>0.100</td><td align='right'>0.030</td><td align='right'>10,790,318</td><td align='right'>0.841</td><td align='right'>73,375</td><td align='right'>19,947</td></tr>
<tr><td>CTRL-rep2</td><td>NA</td><td align='right'>0.973</td><td align='right'>0.880</td><td align='right'>0.093</td><td align='right'>0.027</td><td align='right'>11,531,148</td><td align='right'>0.853</td><td align='right'>73,650</td><td align='right'>19,948</td></tr>
<tr><td>CTRL-rep3</td><td>NA</td><td align='right'>0.972</td><td align='right'>0.853</td><td align='right'>0.120</td><td align='right'>0.027</td><td align='right'>11,156,872</td><td align='right'>0.830</td><td align='right'>70,260</td><td align='right'>18,881</td></tr>
<tr><td>ESRP-DKO-rep1</td><td>NA</td><td align='right'>0.967</td><td align='right'>0.856</td><td align='right'>0.110</td><td align='right'>0.033</td><td align='right'>11,622,814</td><td align='right'>0.820</td><td align='right'>71,088</td><td align='right'>19,258</td></tr>
<tr><td>ESRP-DKO-rep2</td><td>NA</td><td align='right'>0.973</td><td align='right'>0.880</td><td align='right'>0.093</td><td align='right'>0.027</td><td align='right'>12,836,641</td><td align='right'>0.854</td><td align='right'>72,601</td><td align='right'>19,654</td></tr>
<tr><td>ESRP-DKO-rep3</td><td>NA</td><td align='right'>0.961</td><td align='right'>0.866</td><td align='right'>0.095</td><td align='right'>0.039</td><td align='right'>12,225,765</td><td align='right'>0.832</td><td align='right'>72,623</td><td align='right'>19,986</td></tr>
</table>

<p>All of the above rates are per mapped read. <b>Intragenic Rate</b> refers to the fraction of reads  that map within genes (within introns or exons). <b>Exonic Rate</b> is the fraction mapping within exons. <b>Intronic Rate</b> is the fraction mapping within introns. <b>Intergenic Rate</b> is the fraction mapping in the genomic space between genes. <b>Split Reads</b> is the number of reads spanning an exon exon junction.  <b>Expression Profile Efficiency</b> is the ratio of exon reads to total reads. <b>Transcripts/Genes Detected </b> is the number of transcripts/Genes with at least 5 reads. </p>

<h3>Strand Specificity</h3>
<table><tr><th>Sample</th><th>Note</th>
<th>End 1 Sense</th><th>End 1 Antisense</th><th>End 2 Sense</th><th>End 2 Antisense</th><th>End 1 % Sense</th><th>End 2 % Sense</th>
</tr>
<tr><td>CTRL-rep1</td><td>NA</td><td align='right'>191,200</td><td align='right'>15,346,161</td><td align='right'>15,363,625</td><td align='right'>188,034</td><td align='right'>1.231</td><td align='right'>98.791</td></tr>
<tr><td>CTRL-rep2</td><td>NA</td><td align='right'>181,995</td><td align='right'>16,273,929</td><td align='right'>16,289,735</td><td align='right'>179,048</td><td align='right'>1.106</td><td align='right'>98.913</td></tr>
<tr><td>CTRL-rep3</td><td>NA</td><td align='right'>186,481</td><td align='right'>15,896,779</td><td align='right'>15,916,000</td><td align='right'>183,805</td><td align='right'>1.159</td><td align='right'>98.858</td></tr>
<tr><td>ESRP-DKO-rep1</td><td>NA</td><td align='right'>233,099</td><td align='right'>16,365,434</td><td align='right'>16,387,769</td><td align='right'>229,927</td><td align='right'>1.404</td><td align='right'>98.616</td></tr>
<tr><td>ESRP-DKO-rep2</td><td>NA</td><td align='right'>215,894</td><td align='right'>17,903,075</td><td align='right'>17,925,612</td><td align='right'>211,281</td><td align='right'>1.192</td><td align='right'>98.835</td></tr>
<tr><td>ESRP-DKO-rep3</td><td>NA</td><td align='right'>284,964</td><td align='right'>17,037,981</td><td align='right'>17,056,670</td><td align='right'>281,007</td><td align='right'>1.645</td><td align='right'>98.379</td></tr>
</table>

<p><b>End 1/2 Sense</b> are the number of End 1 or 2 reads that were sequenced in the sense direction. Similarly, <b>End 1/2 Antisense</b> are the number of End 1 or 2 reads that were sequenced in the antisense direction.<b>End 1/2 Sense %</b> are percentages of intragenic End 1/2 reads that were sequenced in the sense direction. </p>
