<html>
<head>
  <title>VEP summary</title>
  <script type="text/javascript" src="http://www.google.com/jsapi"></script>
  <script type="text/javascript">
    google.load('visualization', '1', {packages: ['corechart','table']});
  </script>
  <script type="text/javascript">
    
    function init() {
      // charts
      var var_class_pie = drawPie('var_class_pie', 'Variant classes', google.visualization.arrayToDataTable([['Variant class','Count'],['indel',2],['sequence_alteration',51],['substitution',103],['insertion',329],['deletion',401],['SNV',4131]]), null);
var var_class_table = drawTable('var_class_table', 'Variant classes', google.visualization.arrayToDataTable([['Variant class','Count'],['indel',2],['sequence_alteration',51],['substitution',103],['insertion',329],['deletion',401],['SNV',4131]]));

          google.visualization.events.addListener(var_class_pie, 'select', function() {
            var_class_table.setSelection(var_class_pie.getSelection());
          });
          google.visualization.events.addListener(var_class_table, 'select', function() {
            var_class_pie.setSelection(var_class_table.getSelection());
          });
        var var_cons_pie = drawPie('var_cons_pie', 'Consequences (most severe)', google.visualization.arrayToDataTable([['Consequence type','Count'],['splice_donor_variant',2],['splice_acceptor_variant',3],['stop_gained',3],['frameshift_variant',10],['stop_lost',1],['start_lost',2],['inframe_insertion',2],['inframe_deletion',3],['missense_variant',238],['splice_region_variant',64],['incomplete_terminal_codon_variant',3],['synonymous_variant',247],['5_prime_UTR_variant',168],['3_prime_UTR_variant',620],['non_coding_transcript_exon_variant',120],['intron_variant',2432],['upstream_gene_variant',905],['downstream_gene_variant',133],['regulatory_region_variant',18],['intergenic_variant',43]]), {slices: [{color: "#ff7f50"}, {color: "#ff7f50"}, {color: "#ff0000"}, {color: "#ff69b4"}, {color: "#ff0000"}, {}, {color: "#ff69b4"}, {color: "#ff69b4"}, {color: "#ffd700"}, {color: "#ff7f50"}, {color: "#ff00ff"}, {color: "#76ee00"}, {}, {}, {}, {color: "#02599c"}, {color: "#a2b5cd"}, {color: "#a2b5cd"}, {color: "brown"}, {color: "gray"}]});
var var_cons_table = drawTable('var_cons_table', 'Consequences (most severe)', google.visualization.arrayToDataTable([['Consequence type','Count'],['splice_donor_variant',2],['splice_acceptor_variant',3],['stop_gained',3],['frameshift_variant',10],['stop_lost',1],['start_lost',2],['inframe_insertion',2],['inframe_deletion',3],['missense_variant',238],['splice_region_variant',64],['incomplete_terminal_codon_variant',3],['synonymous_variant',247],['5_prime_UTR_variant',168],['3_prime_UTR_variant',620],['non_coding_transcript_exon_variant',120],['intron_variant',2432],['upstream_gene_variant',905],['downstream_gene_variant',133],['regulatory_region_variant',18],['intergenic_variant',43]]));

          google.visualization.events.addListener(var_cons_pie, 'select', function() {
            var_cons_table.setSelection(var_cons_pie.getSelection());
          });
          google.visualization.events.addListener(var_cons_table, 'select', function() {
            var_cons_pie.setSelection(var_cons_table.getSelection());
          });
        var consequences_pie = drawPie('consequences_pie', 'Consequences (all)', google.visualization.arrayToDataTable([['Consequence type','Count'],['splice_acceptor_variant',6],['splice_donor_variant',2],['stop_gained',8],['frameshift_variant',14],['stop_lost',1],['start_lost',2],['inframe_insertion',6],['inframe_deletion',8],['missense_variant',469],['splice_region_variant',221],['incomplete_terminal_codon_variant',3],['synonymous_variant',656],['coding_sequence_variant',5],['5_prime_UTR_variant',312],['3_prime_UTR_variant',1654],['non_coding_transcript_exon_variant',145],['intron_variant',10803],['NMD_transcript_variant',69],['non_coding_transcript_variant',437],['upstream_gene_variant',3455],['downstream_gene_variant',3900],['TF_binding_site_variant',496],['regulatory_region_variant',2257],['intergenic_variant',61]]), {slices: [{color: "#ff7f50"}, {color: "#ff7f50"}, {color: "#ff0000"}, {color: "#ff69b4"}, {color: "#ff0000"}, {}, {color: "#ff69b4"}, {color: "#ff69b4"}, {color: "#ffd700"}, {color: "#ff7f50"}, {color: "#ff00ff"}, {color: "#76ee00"}, {color: "#458b00"}, {}, {}, {}, {color: "#02599c"}, {}, {}, {color: "#a2b5cd"}, {color: "#a2b5cd"}, {}, {color: "brown"}, {color: "gray"}]});
var consequences_table = drawTable('consequences_table', 'Consequences (all)', google.visualization.arrayToDataTable([['Consequence type','Count'],['splice_acceptor_variant',6],['splice_donor_variant',2],['stop_gained',8],['frameshift_variant',14],['stop_lost',1],['start_lost',2],['inframe_insertion',6],['inframe_deletion',8],['missense_variant',469],['splice_region_variant',221],['incomplete_terminal_codon_variant',3],['synonymous_variant',656],['coding_sequence_variant',5],['5_prime_UTR_variant',312],['3_prime_UTR_variant',1654],['non_coding_transcript_exon_variant',145],['intron_variant',10803],['NMD_transcript_variant',69],['non_coding_transcript_variant',437],['upstream_gene_variant',3455],['downstream_gene_variant',3900],['TF_binding_site_variant',496],['regulatory_region_variant',2257],['intergenic_variant',61]]));

          google.visualization.events.addListener(consequences_pie, 'select', function() {
            consequences_table.setSelection(consequences_pie.getSelection());
          });
          google.visualization.events.addListener(consequences_table, 'select', function() {
            consequences_pie.setSelection(consequences_table.getSelection());
          });
        var coding_pie = drawPie('coding_pie', 'Coding consequences', google.visualization.arrayToDataTable([['Consequence type','Count'],['stop_gained',8],['frameshift_variant',14],['stop_lost',1],['start_lost',2],['inframe_insertion',6],['inframe_deletion',8],['missense_variant',469],['incomplete_terminal_codon_variant',3],['synonymous_variant',656],['coding_sequence_variant',5]]), {slices: [{color: "#ff0000"}, {color: "#ff69b4"}, {color: "#ff0000"}, {}, {color: "#ff69b4"}, {color: "#ff69b4"}, {color: "#ffd700"}, {color: "#ff00ff"}, {color: "#76ee00"}, {color: "#458b00"}]});
var coding_table = drawTable('coding_table', 'Coding consequences', google.visualization.arrayToDataTable([['Consequence type','Count'],['stop_gained',8],['frameshift_variant',14],['stop_lost',1],['start_lost',2],['inframe_insertion',6],['inframe_deletion',8],['missense_variant',469],['incomplete_terminal_codon_variant',3],['synonymous_variant',656],['coding_sequence_variant',5]]));

          google.visualization.events.addListener(coding_pie, 'select', function() {
            coding_table.setSelection(coding_pie.getSelection());
          });
          google.visualization.events.addListener(coding_table, 'select', function() {
            coding_pie.setSelection(coding_table.getSelection());
          });
        var sift_pie = drawPie('sift_pie', 'SIFT summary', google.visualization.arrayToDataTable([['Prediction','Count'],['deleterious_low_confidence',30],['tolerated_low_confidence',59],['deleterious',113],['tolerated',250]]), {slices: [{}, {}, {color: "red"}, {color: "green"}]});
var sift_table = drawTable('sift_table', 'SIFT summary', google.visualization.arrayToDataTable([['Prediction','Count'],['deleterious_low_confidence',30],['tolerated_low_confidence',59],['deleterious',113],['tolerated',250]]));

          google.visualization.events.addListener(sift_pie, 'select', function() {
            sift_table.setSelection(sift_pie.getSelection());
          });
          google.visualization.events.addListener(sift_table, 'select', function() {
            sift_pie.setSelection(sift_table.getSelection());
          });
        var polyphen_pie = drawPie('polyphen_pie', 'PolyPhen summary', google.visualization.arrayToDataTable([['Prediction','Count'],['unknown',9],['probably_damaging',46],['possibly_damaging',49],['benign',350]]), {slices: [{color: "blue"}, {}, {}, {color: "green"}]});
var polyphen_table = drawTable('polyphen_table', 'PolyPhen summary', google.visualization.arrayToDataTable([['Prediction','Count'],['unknown',9],['probably_damaging',46],['possibly_damaging',49],['benign',350]]));

          google.visualization.events.addListener(polyphen_pie, 'select', function() {
            polyphen_table.setSelection(polyphen_pie.getSelection());
          });
          google.visualization.events.addListener(polyphen_table, 'select', function() {
            polyphen_pie.setSelection(polyphen_table.getSelection());
          });
        var chr_bar = drawBar('chr_bar', 'Variants by chromosome', google.visualization.arrayToDataTable([['Chromosome','Count'],['chr1',210],['chr10',98],['chr11',130],['chr12',225],['chr13',80],['chr14',1035],['chr15',122],['chr16',170],['chr17',226],['chr18',28],['chr19',148],['chr2',286],['chr20',46],['chr21',104],['chr22',385],['chr3',208],['chr4',127],['chr5',210],['chr6',298],['chr7',437],['chr7_KI270803v1_alt',2],['chr8',65],['chr9',197],['chrX',180]]), {legend: {position: "none"}});
var chr_table = drawTable('chr_table', 'Variants by chromosome', google.visualization.arrayToDataTable([['Chromosome','Count'],['chr1',210],['chr10',98],['chr11',130],['chr12',225],['chr13',80],['chr14',1035],['chr15',122],['chr16',170],['chr17',226],['chr18',28],['chr19',148],['chr2',286],['chr20',46],['chr21',104],['chr22',385],['chr3',208],['chr4',127],['chr5',210],['chr6',298],['chr7',437],['chr7_KI270803v1_alt',2],['chr8',65],['chr9',197],['chrX',180]]));

          google.visualization.events.addListener(chr_bar, 'select', function() {
            chr_table.setSelection(chr_bar.getSelection());
          });
          google.visualization.events.addListener(chr_table, 'select', function() {
            chr_bar.setSelection(chr_table.getSelection());
          });
        var protein_bar = drawBar('protein_bar', 'Position in protein', google.visualization.arrayToDataTable([['Position in protein (percentile)','Count'],['00-10%',101],['10-20%',90],['20-30%',115],['30-40%',83],['40-50%',105],['50-60%',99],['60-70%',121],['70-80%',130],['80-90%',167],['90-100%',156]]), {hAxis: {title: "Position in protein (percentile)", textStyle: {fontSize: 10}}, legend: {position: "none"}});

    }
    
    function drawPie(id, title, data, options) {    
      var pie = new google.visualization.PieChart(document.getElementById(id));
      pie.draw(data, options);
      return pie;
    }
    function drawBar(id, title, data, options) {
      var bar = new google.visualization.ColumnChart(document.getElementById(id));
      bar.draw(data, options);
      return bar;
    }
    function drawTable(id, title, data) {
      var table = new google.visualization.Table(document.getElementById(id));
      table.draw(data, null);
      return table;
    }
    function drawLine(id, title, data, options) {
      var line = new google.visualization.LineChart(document.getElementById(id));
      line.draw(data, options);
      return line;
    }
    function drawArea(id, title, data, options) {
      var area = new google.visualization.AreaChart(document.getElementById(id));
      area.draw(data, options);
      return area;
    }
    google.setOnLoadCallback(init);
  </script>
  
  
  <style type="text/css">
    body {
      font-family: arial, sans-serif;
      margin: 0px;
      padding: 0px;
    }
    
    a {color: #36b;}
    a.visited {color: #006;}
    
    .stats_table {
      margin: 5px;
    }
    
    td {
      padding: 5px;
    }

    th.gradient {
      height: auto;
    }

    .stats_table tr:nth-child(odd) {
      background-color: #f0f0f0;
    }
    
    h3 {
      color: #666;
    }
    
    .masthead {
      background-color: white;
      color: rgb(204, 221, 255);
      height: 80px;
      width: 100%;
      padding: 0px;
    }
    
    .main {
      padding: 10px;
    }
    
    .gradient {
      background: #333366; /* Old browsers */
      background: -moz-linear-gradient(left,  #333366 0%, #ffffff 100%); /* FF3.6+ */
      background: -webkit-gradient(linear, left top, right top, color-stop(0%,#333366), color-stop(100%,#ffffff)); /* Chrome,Safari4+ */
      background: -webkit-linear-gradient(left,  #333366 0%,#ffffff 100%); /* Chrome10+,Safari5.1+ */
      background: -o-linear-gradient(left,  #333366 0%,#ffffff 100%); /* Opera 11.10+ */
      background: -ms-linear-gradient(left,  #333366 0%,#ffffff 100%); /* IE10+ */
      background: linear-gradient(to right,  #333366 0%,#ffffff 100%); /* W3C */
      filter: progid:DXImageTransform.Microsoft.gradient( startColorstr='#333366', endColorstr='#ffffff',GradientType=1 ); /* IE6-9 */
      
      padding: 0px;
      height: 80px;
      width: 700px;
    }
    
    .main_content {
      margin-left: 300px;
    }
    
    .sidemenu {
      width: 260px;
      position: fixed;
      border-style: solid;
      border-width: 2px;
      border-color: rgb(51, 51, 102);
    }
    
    .sidemenu_head {
      width: 250px;
      background-color: rgb(51, 51, 102);
      color: rgb(204, 221, 255);
      padding: 5px;
    }
    
    .sidemenu_body {
      width: 250px;
      padding: 5px;
    }
  </style>
</head>
<body>
<div id="masthead" class="masthead">
  <div style="float: left; display: inline; padding: 10px; height: 80px;">
    <a href="http://www.ensembl.org/"><img src="http://static.ensembl.org/i/e-ensembl.png" width="185" height="45" alt="Ensembl"></a>
  </div>
  
  <div style="float: right; display: inline; height: 80px; background: white; padding: 10px;">
    <a href="http://www.ensembl.org/vep"><img src="http://www.ensembl.org/img/vep_logo.png" width="111" height="50" alt="VEP"></a>
  </div>
  <div class="gradient">
  </div>
</div>
<div class="main">
<div class="sidemenu"><div class="sidemenu_head">Links</div><div class="sidemenu_body"><ul><li><a href="#masthead">Top of page</a></li><li><a href="#run_stats">VEP run statistics</a></li><li><a href="#gen_stats">General statistics</a></li><li><a href="#var_class">Variant classes</a></li><li><a href="#var_cons">Consequences (most severe)</a></li><li><a href="#consequences">Consequences (all)</a></li><li><a href="#coding">Coding consequences</a></li><li><a href="#sift">SIFT summary</a></li><li><a href="#polyphen">PolyPhen summary</a></li><li><a href="#chr">Variants by chromosome</a></li><li><a href="#protein">Position in protein</a></li></ul></div></div><div class='main_content'><h3 id="run_stats">VEP run statistics</h3><table class="stats_table"><tr><td>VEP version (API)</td><td> 103 (103)</td></tr><tr><td>Annotation sources</td><td>Cache: /nobackup/lab_bsf/resources/VEP/homo_sapiens_merged/103_GRCh38; Custom: /nobackup/lab_bsf/resources/ClinVar/hg38/clinvar_20210213.vcf.gz (exact)</td></tr><tr><td>Species</td><td>homo_sapiens</td></tr><tr><td>Command line options</td><td><pre>--allele_number --allow_non_variant --assembly GRCh38 --cache --compress_output bgzip --custom /nobackup/lab_bsf/resources/ClinVar/hg38/clinvar_20210213.vcf.gz,ClnV,vcf,exact,0,ALLELEID,ORIGIN,CLNSIG,CLNREVSTAT,CLNDN --dir_cache /nobackup/lab_bsf/resources/VEP --dir_plugins /nobackup/lab_bsf/resources/VEP/Plugins_103 --dont_skip --everything --exclude_predicted --failed --fasta /nobackup/lab_bsf/resources/VEP/homo_sapiens_merged/103_GRCh38/Homo_sapiens.GRCh38.dna.toplevel.fa.gz --flag_pick_allele_gene --force_overwrite --fork 16 --format vcf --gencode_basic --hgvsg --input_file /nobackup/lab_bsf/users/berguener/junk/BSA_0000_test_variant_calling/cromwell-executions/variant_calling/362b6986-35d9-4d94-9593-8549cc92a6cc/call-somatic_vcf_vep/shard-3/inputs/-164757995/S66731-filtered.vcf.gz --merged --offline --output_file /nobackup/lab_bsf/users/berguener/junk/BSA_0000_test_variant_calling/hg38/mutect2/S66731.vep.vcf.gz --plugin CADD,/nobackup/lab_bsf/resources/CADD/hg38/1.6/whole_genome_SNVs.tsv.gz,/nobackup/lab_bsf/resources/CADD/hg38/1.6/gnomad.genomes.r3.0.indel.tsv.gz --species homo_sapiens --stats_file /nobackup/lab_bsf/users/berguener/junk/BSA_0000_test_variant_calling/hg38/mutect2/S66731.vep.html --tmpdir /nobackup/lab_bsf/users/berguener/junk/BSA_0000_test_variant_calling/hg38/mutect2 --transcript_filter not stable_id match ^X._ --vcf --warning_file /nobackup/lab_bsf/users/berguener/junk/BSA_0000_test_variant_calling/hg38/mutect2/S66731.vep.warning.txt</pre></td></tr><tr><td>Start time</td><td>2021-04-20 18:18:13</td></tr><tr><td>End time</td><td>2021-04-20 18:21:05</td></tr><tr><td>Run time</td><td>172 seconds</td></tr><tr><td>Input file</td><td>/nobackup/lab_bsf/users/berguener/junk/BSA_0000_test_variant_calling/cromwell-executions/variant_calling/362b6986-35d9-4d94-9593-8549cc92a6cc/call-somatic_vcf_vep/shard-3/inputs/-164757995/S66731-filtered.vcf.gz</td></tr><tr><td>Output file</td><td>/nobackup/lab_bsf/users/berguener/junk/BSA_0000_test_variant_calling/hg38/mutect2/S66731.vep.vcf.gz</td></tr></table><h3 id="gen_stats">General statistics</h3><table class="stats_table"><tr><td>Lines of input read</td><td>5017</td></tr><tr><td>Variants processed</td><td>5017</td></tr><tr><td>Variants filtered out</td><td>0</td></tr><tr><td>Novel / existing variants</td><td>412 (8.2) / 4605 (91.8)</td></tr><tr><td>Overlapped genes</td><td>1131</td></tr><tr><td>Overlapped transcripts</td><td>2522</td></tr><tr><td>Overlapped regulatory features</td><td>795</td></tr></table><hr/><h3 id="var_class">Variant classes</h3><div id="var_class_pie" style="width: 800px; height: 200px">&nbsp;</div><div id="var_class_table" style="width: 800px; height: 200px">&nbsp;</div><hr/><h3 id="var_cons">Consequences (most severe)</h3><div id="var_cons_pie" style="width: 800px; height: 400px">&nbsp;</div><div id="var_cons_table" style="width: 800px; height: 200px">&nbsp;</div><hr/><h3 id="consequences">Consequences (all)</h3><div id="consequences_pie" style="width: 800px; height: 400px">&nbsp;</div><div id="consequences_table" style="width: 800px; height: 200px">&nbsp;</div><hr/><h3 id="coding">Coding consequences</h3><div id="coding_pie" style="width: 800px; height: 400px">&nbsp;</div><div id="coding_table" style="width: 800px; height: 200px">&nbsp;</div><hr/><h3 id="sift">SIFT summary</h3><div id="sift_pie" style="width: 800px; height: 200px">&nbsp;</div><div id="sift_table" style="width: 800px; height: 200px">&nbsp;</div><hr/><h3 id="polyphen">PolyPhen summary</h3><div id="polyphen_pie" style="width: 800px; height: 200px">&nbsp;</div><div id="polyphen_table" style="width: 800px; height: 200px">&nbsp;</div><hr/><h3 id="chr">Variants by chromosome</h3><div id="chr_bar" style="width: 800px; height: 200px">&nbsp;</div><div id="chr_table" style="width: 800px; height: 200px">&nbsp;</div><hr/><h3 id="protein">Position in protein</h3><div id="protein_bar" style="width: 800px; height: 200px">&nbsp;</div></div>
</div></body>
</html>
